src/OpenTelemetry.Api/Trace/TracerNew.cs

```diff
@@ -127,6 +129,7 @@ public TelemetrySpanNew StartSpan(string name, SpanKind kind, in TelemetrySpanNe
         /// <param name="startTime"> Start time for the span.</param>
         /// <returns>Span instance.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("ApiDesign", "RS0026:Do not add multiple public overloads with optional parameters", Justification = "Temp justification")]
```

Comment:
@cijothomas , i had to include this suppression

Reply:
```suggestion
        [System.Diagnostics.CodeAnalysis.SuppressMessage("ApiDesign", "RS0026:Do not add multiple public overloads with optional parameters", Justification = "This was agreed on to be the friendliest API surface")]
```